lib/galaxy/tools/actions/upload_common.py

```diff
@@ -26,6 +26,8 @@ def validate_url(url, ip_whitelist):
     if not (url.lstrip().startswith('http://') or url.lstrip().startswith('https://')):
         return url
 
+    # Strip leading whitespace
+    url = url.lstrip()
```

Comment:
should this be just `url = url.strip()`, since whitespace on either side is invalid?